src/framework/uicomponents/qml/Muse/UiComponents/internal/StyledMenu.qml

```diff
@@ -32,6 +32,9 @@ MenuView {
 
     property alias model: view.model
 
+    property NavigationSection notationViewNavigationSection: null
```

Comment:
The `UiComponents` module should not contain the word "notation", because it is intended to be as general as possible. Something like `navigationSectionOverride` would be okay though.